README.rst

```diff
@@ -308,6 +308,14 @@ seems to be the complete history for each conversation and nothing gets truncate
 
 Unfortunately, this cannot be remedied unless Facebook fixes the problem on their end.
 
+Why are repeated names not showing?
+----------------------------------------
```

Comment:
PyPi's RST parser is extremely sensitive. To ensure this still renders, make sure the number of hyphens matches the number if characters above it.